scss/components/_activities-calendar.scss

```diff
@@ -82,6 +82,8 @@
 
   // week days header
   tr:nth-child(2n) {
+    border-top-left-radius: 5px;
```

Comment:
can we put `5px` in a variable? This needs to be defined as a rule, not an exception.